README.md

```diff
@@ -25,22 +25,15 @@ github.com/java-operator-sdk/kubebuilder-plugin => /Users/sushah/go/src/github.c
 javav1 "github.com/java-operator-sdk/kubebuilder-plugin/pkg/java/v1"
 ```
 
-- Introduce the java bundle in `GetPluginsCLIAndRoot()` method. 
-```
-javaBundle, _ := plugin.NewBundle("quarkus"+plugins.DefaultNameQualifier, plugin.Version{Number: 1},
-		&javav1.Plugin{},
-	)
-```
-
-- Add the created javaBundle to the `cli.New`
+- Add the java Plugin to the `cli.New`
 
 ```
     cli.WithPlugins(
 			ansibleBundle,
 			gov2Bundle,
 			gov3Bundle,
 			helmBundle,
-			javaBundle,
+			&javav1.Plugin{},
```

Comment:
That's not what I was trying to say. I was saying that we need to define the bundle in the plugin project. And then in the SDK we instantiate the plugin.

So the SDK would look like this

```golang
import javav1 "github.com/java-operator-sdk/kubebuilder-plugin/pkg/java/v1"
cli.WithPlugins(
    ansibleBundle,
    ...
    javav1.NewJavaBundle(),
),
```

But that `JavaBundle` *needs* to be defined in the java plugin project. So in  `pkg/java/v1/bundle.go` I'd have a 

```golang
...
func NewJavaBundle() plugin.Bundle {
    javaBundle, _ := plugin.NewBundle("quarkus"+plugins.DefaultNameQualifier, 
		plugin.Version{Number: 1},
		&Plugin{},
	)
    return javaBundle
}
```


```

Reply:
Ah! okay got it!

Reply:
But, it is again part of **internal**. I need to copy all files from 	
**"github.com/operator-framework/operator-sdk/internal/plugins"** 
I will add those changes in our java kubebuilder plugin

Reply:
Hrm maybe we don't want to do this then. I didn't think we had to copy anything to accomplish that. If so then let's not bother, it's no worth copying